README.rst

```diff
@@ -99,13 +100,17 @@ iEEG is experimental at the moment.
 Command Line Interface
 ----------------------
 
-In addition to ``import mne_bids``\ , you can use the command line interface.
+In addition to ``import mne_bids``, you can use the command line interface.
+Simply type ``mne_bids`` in your command line and press enter, to see the
+accepted commands. Then type ``mne_bids <command> --help``, where ``<command>``
+is one of the accepted commands, to get more information about that
+``<command>``.
 
 Example:
 
 .. code-block:: bash
 
-   $ mne_bids raw_to_bids --subject_id sub01 --task rest --raw data.edf --output_path new_path
+   $ mne_bids raw_to_bids --help
```

Comment:
if you can put a few usage examples it will help a lot. It should be simpler to adapt and use than reading the --help